docs/_static/nuts-consent-logic.yaml

```diff
@@ -77,17 +77,12 @@ components:
         - records
       properties:
         subject:
-          $ref: "#/components/schemas/SubjectURI"
+          $ref: "#/components/schemas/IdentifierURI"
         custodian:
-          $ref: "#/components/schemas/CustodianURI"
+          $ref: "#/components/schemas/IdentifierURI"
         actor:
-          $ref: "#/components/schemas/ActorURI"
+          $ref: "#/components/schemas/IdentifierURI"
```

Comment:
The disadvantage of this, is that the examples will look less specific in the Swagger API explorer: https://nuts-documentation.readthedocs.io/en/v0.14.0/pages/API/consent-logic.html

Reply:
Discussed on Slack, if we keep 4 different types for the same data type we are generating a more complex API with a lot of boilerplate code. We'll refer to the documentation on what OIDs to use instead.